docs/adrs/003-Retry_endpoint_design.md

```diff
@@ -1,11 +1,11 @@
-# Retry Endpoint
+# Retry Endpoint Design (Round 2!)
 
 Date: 2021-05-19
 
 
 ## Motivation for the Endpoint:
 
-The current requirement for the Batch/Bulk UI is to be able to retry jobs that have either "errored" out, or were cancelled.
+The current requirement for the Batch/Bulk UI is to be able to retry jobs that have either "errored" out, or were terminated.
 The UI allows you to retry either single jobs, or multiple jobs, and saves you from having to cancel and resubmit each job individually,
 which is not really possibly with the UI anyway.
 
```

Comment:
comment for lines 13 - 20: can we change this to being quoted text, rather than being styled as code?

See https://docs.github.com/en/github/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#quoting-text

Reply:
Comment for line 26: rephrase to

> The endpoint takes a job or list of job ids and attempts to resubmit them to the queue, using the same parameters and version of the app as the original job.

Reply:
done